modules/disable-quickstarts-admin-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="disable-quickstarts-admin-console_{context}"]
+= Disabling quick starts in the web console
+
+You can use the *Administrator* perspective of the web console to disable one or more quick starts.
+
+.Prerequisites
+
+* You have cluster administrator permissions and are logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administation* -> *Cluster Settings*.
+. In the *Cluster Settings* page, click the *Configuration* tab.
+. In the *Configuration* page, click the *Console* configuration resource with the description *operator.openshift.io*.
++
+image::cluster-settings-console.png[Image of the Configuration page showing the correct Console configuration resource to select]
+
+. In the *Action* drop-down list, click *Customize*. This takes you to the *Cluster configuration* page.
++
+image::cluster-configuration-general-tab.png[Cluster configuration page open on the General tab]
+
+. In the *General* tab, under *Quick starts*, you can select items in either the *Enabled* or *Disabled* list, and move them from one list to the other by using the arrow buttons.
+
+** To enable or disable a single quick start, click on the quick start, then use the single arrow buttons to move the quick start to the appropriate list.
+** To enable or disable multiple quick starts at once, you can hold Ctrl and click the quick starts you want to move, before using the single arrow buttons to move the quick starts to the appropriate list.
```

Comment:
```suggestion
** To enable or disable multiple quick starts at once, press Ctrl and click the quick starts you want to move. Then, use the single arrow buttons to move the quick starts to the appropriate list.
```
- Because this is part of a procedure, consider using the imperative rather than "you can".
- Per ISG, use "press" with keyboard function keys like Ctrl: https://www.ibm.com/docs/en/ibm-style?topic=elements-keyboard-keys#verbs-with-keyboard-keys
- Consider splitting this one long sentence into two.